plugins/janus_videoroom.c

```diff
@@ -2292,6 +2269,36 @@ void janus_videoroom_incoming_rtp(janus_plugin_session *handle, int video, char
 	if(!session || session->destroyed || !session->participant || session->participant_type != janus_videoroom_p_type_publisher)
 		return;
 	janus_videoroom_participant *participant = (janus_videoroom_participant *)session->participant;
+	if(participant->audio_active) {
```

Comment:
I believe all this should be optional, that is, when you create a room you should be able to decide whether you want to be notified about who's speaking or not. The way it works now, this seems to be automatically done when the extension is found, am I correct?